printer_mods/arhodiewithsoul/Electronics_Brackets/Rear_Electronics_Enclosure/Unsupported_Controllers/readme.md

```diff
@@ -0,0 +1,5 @@
+Contains various mounts for Unsupported Controllers
+
+12v Ramps, 12v Mosfets, Fly Mini v1, TwoTrees SKR E3 DIP etc...
+
+![ramps_mount_flipped (Large)](https://user-images.githubusercontent.com/80538348/116250259-7916b980-a765-11eb-9fdb-89412e6b3a47.jpg)
```

Comment:
`ramps_mount_flipped.stl` has too steep of an overhang under the cutout, maybe you can just replace those with 45 degree chamfers

<img width="380" alt="Screen Shot 2021-05-02 at 17 33 06" src="https://user-images.githubusercontent.com/69389593/116828442-a8b23100-ab6c-11eb-9c7d-a164b334da06.png">

Reply:
Updated part